db/migrate/20151102115238_add_email_to_doctors.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToDoctors < ActiveRecord::Migration
+  def change
+    add_column :doctors, :email, :string
```

Comment:
Ok. I understand now. But I have create appointment in another migration. I will do as your suggestion next time. thanks